modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ToggleTag.java

```diff
@@ -159,9 +161,13 @@ protected String getHydratedModuleName() {
 	@Override
 	protected Map<String, Object> prepareProps(Map<String, Object> props) {
 		props.put("disabled", _disabled);
-		props.put("helpText", _helpText);
+		props.put("helpText", LanguageUtil.get(
+			TagResourceBundleUtil.getResourceBundle(pageContext), _helpText));
 		props.put("id", _id);
-		props.put("label", _label);
+		props.put(
+			"label",
+			LanguageUtil.get(
+				TagResourceBundleUtil.getResourceBundle(pageContext), _label));
 		props.put("labelOff", _labelOff);
 		props.put("labelOn", _labelOn);
```

Comment:
shouldn't we apply `LanguageUtil.get()` to `_labelOff` and `labelOn` when setting the props?

Reply:
Good catch! I'll update this.